app/helpers/users_helper.rb

```diff
@@ -0,0 +1,5 @@
+module UsersHelper
+  def available_times_of_day
+    (0...24).map { |hour| Time.utc(2000, 1, 1, hour, 00, 0) }
```

Comment:
To make it easier for the user, I suggest that we take the user timezone also into consideration.